dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -3066,6 +3068,23 @@ public void testCursorPreviousWithUpdates() {
         }
     }
 
+    @Test //TODO
+    @Ignore("Reference issue: https://github.com/OpenLiberty/open-liberty/issues/29475")
+    public void testFetchTypeDefault() {
+        ratings.clear();
+
+        Rating.Reviewer user1 = new Rating.Reviewer("Rex", "TestRecordWithEmbeddables", "rex@openliberty.io");
+        Rating.Item toaster = new Rating.Item("toaster", 28.98f);
+        Set<String> comments = Set.of("Burns everything.", "Often gets stuck.", "Bagels don't fit.");
+
+        ratings.add(new Rating(1000, toaster, 2, user1, comments));
+
+        Rating user1Rating = ratings.get(1000).orElseThrow();
+
+        assertFalse("Expected comments to be populated when using fetch type eager", user1Rating.comments().isEmpty());
+        assertTrue("Expected comments to be populated when using fetch type eager", user1Rating.comments().containsAll(comments));
```

Comment:
This will display more detail if it ever fails and is also a more accurate comparison,
```suggestion
        assertEquals("Expected comments to be populated when using fetch type eager", comments, user1Rating.comments());
```